build-info-extractor-gradle/src/main/groovy/org/jfrog/gradle/plugin/artifactory/task/DistributeTask.java

```diff
@@ -0,0 +1,75 @@
+package org.jfrog.gradle.plugin.artifactory.task;
+
+import org.apache.commons.lang.StringUtils;
+import org.apache.http.HttpResponse;
+import org.apache.http.StatusLine;
+import org.apache.http.util.EntityUtils;
+import org.gradle.api.DefaultTask;
+import org.gradle.api.logging.Logger;
+import org.gradle.api.logging.Logging;
+import org.gradle.api.tasks.TaskAction;
+import org.jfrog.build.api.release.Distribution;
+import org.jfrog.build.extractor.clientConfiguration.ArtifactoryClientConfiguration;
+import org.jfrog.build.extractor.clientConfiguration.client.ArtifactoryBuildInfoClient;
+import org.jfrog.gradle.plugin.artifactory.dsl.ArtifactoryPluginConvention;
+import org.jfrog.gradle.plugin.artifactory.dsl.DistributerConfig;
+import org.jfrog.gradle.plugin.artifactory.extractor.GradleClientLogger;
+import java.io.IOException;
+import java.util.ArrayList;
+
+public class DistributeTask extends DefaultTask {
+
+    public static final String DISTRIBUTE_TASK_NAME = "artifactoryDistribute";
```

Comment:
A thought: Users that used the artifactoryDeploy task could use `gradle aD`. Now we have 2 tasks with `aD`.

Reply:
We can use "artifactoryDistributeBuild"

Reply:
Changed to artifactoryDistributeBuild.